autoendpoint/src/extractors/notification_headers.rs

```diff
@@ -411,5 +423,31 @@ mod tests {
         );
     }
 
+    /// The encryption and crypto-key headers are stripped of Base64 padding and
+    /// double-quotes.
+    #[test]
+    fn strip_headers() {
+        let req = TestRequest::post()
+            .header("TTL", "10")
+            .header("Content-Encoding", "aesgcm")
+            .header("Encryption", "salt=\"foo\"")
+            .header("Crypto-Key", "dh=\"deadbeef==\"")
```

Comment:
I don't think you'll have a problem here, but could you add the subcomponent to this?

(e.g.  `.header("Crypto-Key", "keyid=\"p256dh\";dh=\"deadbeef==\""_)` )

I'd like to make sure that future fixes break if they can't parse that too.